include/aws/common/byte_buf.h

```diff
@@ -624,24 +640,24 @@ AWS_COMMON_API bool aws_byte_cursor_read_be24(struct aws_byte_cursor *cur, uint3
 AWS_COMMON_API bool aws_byte_cursor_read_be32(struct aws_byte_cursor *cur, uint32_t *var);
 
 /**
- * Reads a 32-bit value in network byte order from cur, and places it in host
+ * Reads a 64-bit value in network byte order from cur, and places it in host
  * byte order into var.
  *
  * On success, returns true and updates the cursor pointer/length accordingly.
  * If there is insufficient space in the cursor, returns false, leaving the
  * cursor unchanged.
  */
-AWS_COMMON_API bool aws_byte_cursor_read_float_be32(struct aws_byte_cursor *cur, float *var);
+AWS_COMMON_API bool aws_byte_cursor_read_be64(struct aws_byte_cursor *cur, uint64_t *var);
 
 /**
- * Reads a 64-bit value in network byte order from cur, and places it in host
+ * Reads a 32-bit value in network byte order from cur, and places it in host
  * byte order into var.
  *
  * On success, returns true and updates the cursor pointer/length accordingly.
  * If there is insufficient space in the cursor, returns false, leaving the
  * cursor unchanged.
  */
-AWS_COMMON_API bool aws_byte_cursor_read_float_be64(struct aws_byte_cursor *cur, double *var);
+AWS_COMMON_API bool aws_byte_cursor_read_float_be32(struct aws_byte_cursor *cur, float *var);
```

Comment:
These didn't really change names. I just moved them so all the whole-number `read` functions were together and all the `read_float` functions were together